packages/coinstac-ui/app/render/components/activity-indicator/activity-indicator.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+import './activity-indicator.css';
+
+export default class ActivityIndicator extends React.Component {
```

Comment:
My general thinking with this PR is that it's easy to swap our implementation for something else (like [Material Design's “Progress Indicators”](https://material.io/design/components/progress-indicators.html)). It also cleans up the `App` component.

Reply:
👍